### PR TITLE
bpo-32411: IDLE: Remove line number sort in browser.py

### PR DESCRIPTION
**Pull request by csabella**
Since the input dictionary is created in line number and dictionaries are guaranteed to be in insertion order, the sort is no longer needed.

https://bugs.python.org/issue32411

### CODE REVIEW

File: Misc/NEWS.d/next/IDLE/2017-12-25-18-48-50.bpo-32411.vNwDhe.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+In browser.py, remove extraneous sorting by line number since dictionary was
```
Comment (merwok) [inline Misc/NEWS.d/next/IDLE/2017-12-25-18-48-50.bpo-32411.vNwDhe.rst]: `browser` is not a top-level module, I think this entry should mention `idlelib.browser` or `the IDLE xyz plugin`.

**Review by terryjreedy — CHANGES_REQUESTED** [review]
The requested 'change' is to wait for circumstances to change ;-).

**Comment by rhettinger** [comment]
LGTM.

**Review by terryjreedy — APPROVED** [review]

**Comment by csabella** [comment]
@terryjreedy, thank you!

**Pull request merged**